services/jetbrains-space.js

```diff
@@ -0,0 +1,20 @@
+// https://jetbrains.com/help/space/automation-environment-variables.html#automation
+const {parseBranch} = require('../lib/utils.js');
+
+module.exports = {
+  detect({env}) {
+    return Boolean(env.JB_SPACE_EXECUTION_NUMBER);
+  },
+  configuration({env}) {
+    const projectKey = env.JB_SPACE_PROJECT_KEY?.toLowerCase();
```

Comment:
@123FLO321 it looks like the `?.` broke the build for v12.20. i'm not sure why that didnt block the auto merge from happening.

since we havent officially dropped support for that version yet, we need to handle that for now until we release a breaking change that drops support for v12 (we should do that anyway since it is EOL). could you provide an update that would be compatible with node v12.20, or should we back this change out until we drop v12 support?